index.md

```diff
@@ -19,13 +19,28 @@ Data Carpentry's aim is to teach researchers basic concepts, skills, and tools f
 {: .prereq}
 
 > ## Data
->
-> The data for this workshop are is the [FIXME](FIXME) available on FigShare, with a CC-BY license available for reuse.
->
-> FIXME add a short description about the dataset.
->
+> Data for this lesson comes from the National Ecological Observatory Network. These data files represent teaching version of the data, 
+> with sufficient complexity to teach many aspects of  data analysis and management, but with many complexities removed to allow students
+> to focus on the core ideas and skills being taught.  
+> 
+> There are four data sets included, all of which are available 
+> [on Figshare](https://figshare.com/articles/Spatio_temporal_Series_Teaching_Data_Subsets/2009586)
+> under a CC-BY license. You can download all of the data used in this workshop by clicking 
+> [this download link](https://ndownloader.figshare.com/articles/2009586/versions/10). 
+> Clicking the download link will automatically download all of the files to your default download directory as a single compressed
+> (`.zip`) file. To expand this file, double click the folder icon in your file navigator application (for Macs, this is the Finder 
+> application).
+> 
+> | Dataset | File name | Description |
+> | ---- | ------| ---- | 
+> | Site layout shapefiles | NEON-DS-Site-Layout-Files.zip | A set of shapefiles for the NEON's Harvard Forest field site and US and (some) state boundary layers. | 
+> | Meteorological data |  NEON-DS-Met-Time-Series.zip | Precipitation, temperature and other variables collected from a flux tower at the NEON Harvard Forest site
+> | Airborne remote sensing data | NEON-DS-Airborne-RemoteSensing.zip | LiDAR data including a canopy height model, digital elevation model and digital surface model for NEON's Harvard Forest and San Joaquin Experimental Range field sites. 
+> | Landstat NDVI raster data | NEON-DS-Landsat-NDVI.zip |  2011 NDVI data product provided by USGS cropped to NEON's Harvard Forest and San Joaquin Experimental Range field sites. | 
```

Comment:
How about:

| Landsat 7 NDVI raster data | NEON-DS-Landsat-NDVI.zip |  2011 NDVI data product derived from Landsat 7 and processed by USGS cropped to NEON's Harvard Forest and San Joaquin Experimental Range field sites.